src/twisted/conch/test/keydata.py

```diff
@@ -248,27 +248,31 @@
 # following it. It is not any standard key format and was probably a bug in
 # OpenSSH at some point.
 privateRSA_openssh_alternate = b"""-----BEGIN RSA PRIVATE KEY-----
-MIIEqTCCBKMCAQACggEBANVqrHgj1tYb7CWhUMR3Y1CERQFVQhQqKuDQYO7U6aOtSvo5Bl6EVXVf
```

Comment:
The comment above this private key explains that it's wrapped in an extra sequence and ends with an empty one.  Here's the diff between the old and new keys in [der-ascii](https://github.com/google/der-ascii) format:

```diff
--- bad.ascii	2020-07-18 13:16:34.440366325 -0700
+++ good.ascii	2020-07-18 13:16:28.080343903 -0700
@@ -8,7 +8,7 @@
     INTEGER { `00fb44178ba436be1e371da7f6616c04c4aadd783e078c1e3302ae031487837ae59e7d0867a8f2aabf1270cf72a9a7c70b1d88d520fd9c63ca4730554e8bc4cff47f16a4921274a109c2c46e9c8c33efa5e5f7e02bad4f5c11aa1a84375bfd7aeac3cd7cb0c8e41f5463b5c7afdff409a7fcc725fc5ce991d792c5981e56d3b123` }
     INTEGER { `51afe9929f940b4a84653e94b33b9210b574b88cc9efc90e0132fa2f6812a10326aecf5168144c269b28a40233085fe1a770980134795e528ec4cf36bc1f4b55acebc15384c7e161a84ad4a2ff400d801f12a950c02a1875940c069b1650a84bec41cd7beff74bc9750268c498b88688185a43e7023397226493830c452a7ced` }
     INTEGER { `696637c640dd21c504f3b0b83e47947c76fc2deb3f393c95c3430151c44297f3e816a4c6eeecd44910503804ee3bcdd7d0cccc32699007a01b5a9ffe31cd1e3a7e711e19941a4e4f0cdf5f52d5d1176eecd79cb6559ddb8e215fbc2488b6fcafabf0efa42cb0dc9f86b00312b88be2dd0fc0ee233d4a50fe2fb8294658bfb627` }
-    INTEGER { `00854b1b7a9b1210379e1fad5edafec696fedf356bb934e21697922609bdbd702003a735bd2d1ba0d207472bd4dea8a807071bb80420a727413c6c3939e941cee7171dd14c5cbc3dd22626fe6ad6fd4872ae46fa7bc3d31960441da513a780f56329d47a5d0607165df68b3dcb643ae2845a4d8c062d2d9d1ceb834c783d7954ce` }
+    INTEGER { `6616f92034d05425bc6131ac82fbef9c351e7e4a550268950d0a93be1ebfe140fa90a7547031e978fce066b722779aebddd5d23046cae2d75e419bcfb5483551aa25dcdedb34294821dbd674fc7ee52f53f79c0970b9e4a076a6ad7a7422349c4f17cb3fe0aae55ea527de3fa57f0fa688f515d65f371792e0d60549103b92f6` }
   }
   SEQUENCE {}
 }
```
This shows an empty sequence at the end, and shows that the outer sequence didn't change. :+1: